Functions.Templates/Templates/BlobTrigger-FSharp-Isolated/.template.config/template.json

```diff
@@ -46,7 +46,7 @@
             "args": {
```

Comment:
Same comment here about updating the desc to say `Microsoft.Azure.Functions.Worker.Extensions.Storage.Blobs`